heimdall-gateway/pom.xml

```diff
@@ -115,6 +115,29 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		
+		<dependency>
+		    <groupId>org.hibernate.validator</groupId>
+		    <artifactId>hibernate-validator</artifactId>
+		    <version>6.0.2.Final</version>
+		</dependency>
+		<dependency>
+		    <groupId>org.hibernate.validator</groupId>
+		    <artifactId>hibernate-validator-annotation-processor</artifactId>
+		    <version>6.0.2.Final</version>
+		</dependency>
+		
+		<dependency>
+		    <groupId>javax.el</groupId>
+		    <artifactId>javax.el-api</artifactId>
+		    <version>3.0.0</version>
+		</dependency>
+		 
+		<dependency>
+		    <groupId>org.glassfish.web</groupId>
```

Comment:
this dependency it's really necessary?

Reply:
Hey @MAAARKIN I did some tests and you are right! Apparently the tomcat embedded provide the implementation of variable interpolation and expression.